File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long rialnum= x;
        long long revnum= 0;
        while (x>0){
            int lastdigit= x%10;
            revnum= (revnum*10)+lastdigit;
            x= x/10;
        }
        if (revnum==rialnum){
            return true;
        }
        return false;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        ini_x= x
        rev_x= 0
        while x>0:
            lastdigit= x%10
            rev_x= (rev_x*10)+lastdigit
            x= x//10
        if (ini_x==rev_x):
            return True
        return False